filebeat/filebeat.reference.yml

```
@@ -933,7 +933,7 @@ filebeat.inputs:
   # List of hosts to pool to retrieve the slow log information.
   #hosts: ["localhost:6379"]
 
-  # How often the input checks for redis slow log.
+  # How often do the input checks for redis slow log.
```

```suggestion
  # How often the input checks for redis slow log.
```
I think the original is okay as is.